presto-docs/src/main/sphinx/security/built-in-system-access-control.rst

```diff
@@ -169,14 +173,65 @@ following rules:
 For group-based rules to match, users need to be assigned to groups by a
 :doc:`/develop/group-provider`.
 
+.. _schema_rules:
+
+Schema Rules
+------------
+
+These rules allow you to grant ownership of a schema. Having ownership of an
+schema allows users to execute ``DROP SCHEMA``, ``ALTER SCHEMA`` (both renaming
+and setting authorization), ``SHOW CREATE SCHEMA``. Each rule is composed of
+the following fields:
+
+* ``user`` (optional): regex to match against user name. Defaults to ``.*``.
+* ``group`` (optional): regex to match against group names. Defaults to ``.*``.
+* ``schema`` (optional): regex to match against schema names. Defaults to ``.*``.
+* ``owner`` (required): boolean indicating whether the user is to be considered
+  an owner of the schema. Defaults to ``false``.
+
+For example, to provide ownership of all schemas to user ``admin``, treat all
+users as owners of ``default`` schema and prevent user ``guest`` from ownership
+of any schema, you can use the following rules:
+
+.. literalinclude:: schema-access.json
+    :language: json
+
+.. _table_rules:
+
+Table Rules
+-----------
+
+These rules define the privileges for table access for users. If no table rules
+are specified, all users are treated as having all privileges by default. Each
+rule is composed of the following fields:
+
+* ``user`` (optional): regex to match against user name. Defaults to ``.*``.
+* ``group`` (optional): regex to match against group names. Defaults to ``.*``.
+* ``schema`` (optional): regex to match against schema names. Defaults to ``.*``.
+* ``table`` (optional): regex to match against table names. Defaults to ``.*``.
+* ``privileges`` (required): zero or more of ``SELECT``, ``INSERT``,
+  ``DELETE``, ``OWNERSHIP``, ``GRANT_SELECT``
+
+.. note::
+
+    Users always have all privileges on all tables in the ``information_schema``.
```

Comment:
This seems misleading, since metadata access like `information_schema` has its own security restrictions. Perhaps this should say
```
.. note:: These rules do not apply to ``information_schema``.